cli/js/files.ts

```diff
@@ -20,25 +20,73 @@ import {
 /** Open a file and return an instance of the `File` object
  *  synchronously.
  *
- *       const file = Deno.openSync("/foo/bar.txt");
+ *       const file = Deno.openSync("/foo/bar.txt", { read: true });
+ */
+export function openSync(filename: string, capability?: OpenCapability): File;
```

Comment:
I don't like the name capability. I prefer "mode". Looking through the comments I see that's what was done originally.

Reply:
Yes it was. For me personally more important than name would be good docs, but in the other hand two different names suggest that they're doing different things, which they don't